test/python/taskbasedsimulator/test_path_sim_qasm_simulator.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+
+from qiskit import QuantumCircuit, BasicAer
+from jkq.ddsim.pathqasmsimulator import PathQasmSimulator
+from qiskit import execute
+
+
+class JKQQasmSimulatorTest(unittest.TestCase):
+    """Runs backend checks and the Basic qasm_simulator tests from Qiskit Terra."""
+
+    def setUp(self):
+        self.backend = PathQasmSimulator()
+        self.circuit = QuantumCircuit.from_qasm_str('''OPENQASM 2.0;
+            include "qelib1.inc";
+            qreg q[3];
+            qreg r[3];
+            h q;
+            cx q, r;
+            creg c[3];
+            creg d[3];
+            barrier q;
+            measure q->c;''')
```

Comment:
That's still not the same test as in the other files...

Reply:
Do you mean literally the same test?

Reply:
See https://github.com/iic-jku/ddsim/blob/22e7aece7ce66f1fa267d60528fa0323c9d50ad2/test/python/simulator/test_qasm_simulator.py#L12-L24